integration/openshift/README.md

```diff
@@ -0,0 +1,27 @@
+# Openshift integration with Clear Containers
+
+In this section you can find scripts to setup an Openshift enviornment
+on top of Clear Containers. Currently, these scripts only work for **Fedora**.
+
+Please execute the steps below on a Clean Environment.
+
+Setup the environment:
+
+```
+$ ./setup.sh
+```
+
+Initialize the Master server and the node server:
+
+```
+$ ./init.sh
+```
+
+You can also find a basic test which can be executed in order to verify that the
+environment was build correctly. To run the test you need the `bats` framework.
```

Comment:
I suggest we re-write the sentence beginning on line 20:
Run a basic verification test to check the environment was built correctly (recommended).  To run the test you need the `bats` framework.